src/widget/woverviewrgb.cpp

```diff
@@ -60,10 +60,23 @@ bool WOverviewRGB::drawNextPixmapPart() {
 
     QColor color;
 
-    unsigned char low, mid, high;
+    qreal lowColor_r, lowColor_g, lowColor_b;
+    m_signalColors.getRgbLowColor().getRgbF(&lowColor_r, &lowColor_g, &lowColor_b);
+
+    qreal midColor_r, midColor_g, midColor_b;
+    m_signalColors.getRgbMidColor().getRgbF(&midColor_r, &midColor_g, &midColor_b);
+
+    qreal highColor_r, highColor_g, highColor_b;
+    m_signalColors.getRgbHighColor().getRgbF(&highColor_r, &highColor_g, &highColor_b);
+
+    // "Raw" LMH values
```

Comment:
About that -- my intention was to make the code inside the big loop symmetrical i.e. code for left channel to be as similar as possible to code for right channel.